[2022-12-11 19:44:38,294] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:44:18.299689+00:00 [queued]>
[2022-12-11 19:44:38,363] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:44:18.299689+00:00 [queued]>
[2022-12-11 19:44:38,364] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:44:38,366] {taskinstance.py:1357} INFO - Starting attempt 1 of 3
[2022-12-11 19:44:38,367] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:44:38,476] {taskinstance.py:1377} INFO - Executing <Task(PythonSensor): wait_for_file> on 2022-12-11 19:44:18.299689+00:00
[2022-12-11 19:44:38,513] {standard_task_runner.py:52} INFO - Started process 5128 to run task
[2022-12-11 19:44:38,563] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'wait_for_file', 'manual__2022-12-11T19:44:18.299689+00:00', '--job-id', '1274', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpazphu2l5', '--error-file', '/tmp/tmptf74zi9e']
[2022-12-11 19:44:38,568] {standard_task_runner.py:80} INFO - Job 1274: Subtask wait_for_file
[2022-12-11 19:44:38,823] {task_command.py:369} INFO - Running <TaskInstance: first_dag.wait_for_file manual__2022-12-11T19:44:18.299689+00:00 [running]> on host 7e39590a7807
[2022-12-11 19:44:39,360] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=wait_for_file
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T19:44:18.299689+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T19:44:18.299689+00:00
[2022-12-11 19:44:39,364] {python.py:66} INFO - Poking callable: <function _wait_for_file at 0x7f6cec7af200>
[2022-12-11 19:44:39,415] {logging_mixin.py:115} INFO - _wait_for_file True
[2022-12-11 19:44:39,418] {logging_mixin.py:115} INFO - total attempt 0
[2022-12-11 19:44:59,118] {python.py:66} INFO - Poking callable: <function _wait_for_file at 0x7f6cec7af200>
[2022-12-11 19:44:59,144] {logging_mixin.py:115} INFO - _wait_for_file True
[2022-12-11 19:44:59,146] {logging_mixin.py:115} INFO - total attempt 0
[2022-12-11 19:44:59,149] {base.py:301} INFO - Success criteria met. Exiting.
[2022-12-11 19:44:59,186] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=wait_for_file, execution_date=20221211T194418, start_date=20221211T194438, end_date=20221211T194459
[2022-12-11 19:44:59,459] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:44:59,529] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
